.github/workflows/release.yml

```diff
@@ -0,0 +1,245 @@
+on:
+  push:
+    tags:
+      - 'v[0-9].*' # Release tags matching v*, i.e. v1.0, v20.15.10
+
+name: Release
+
+jobs:
+
+  create_release:
+    name: Create release
+    if: >
+      github.repository_owner == 'input-output-hk'
+      || startsWith(github.ref, 'refs/heads/ci/test/')
+      || startsWith(github.ref, 'refs/tags/') && contains(github.ref, '-ci-test.')
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.release_info.outputs.version }}
+      tag: ${{ steps.release_info.outputs.tag }}
+      date: ${{ steps.release_info.outputs.date }}
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          submodules: 'true'
+
+      - id: release_info
+        name: Get release information
+        run: python3 ci/release-info.py "$GITHUB_EVENT_NAME"
```

Comment:
I think this file is missing, and there is some hard-coded stuff there, like the workspaces in jormungandr